dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt

```diff
@@ -0,0 +1,181 @@
+            clientToken,
+            DatadogEndpoint.TRACES_US,
+            DEFAULT_SERVICE_NAME
+        )
+        private var crashReportConfig: FeatureConfig = FeatureConfig(
+            clientToken,
+            DatadogEndpoint.LOGS_US,
+            DEFAULT_SERVICE_NAME
+        )
+
+        private var logsEnabled: Boolean = true
+        private var tracesEnabled: Boolean = true
+        private var crashReportsEnabled: Boolean = true
+        private var needsClearTextHttp: Boolean = false
+
+        private var serviceName: String = DEFAULT_SERVICE_NAME
+
+        /**
+         * Builds a [DatadogConfig] based on the current state of this Builder.
+         */
+        fun build(): DatadogConfig {
+
+            return DatadogConfig(
+                needsClearTextHttp = needsClearTextHttp,
+                logsConfig = if (logsEnabled) logsConfig else null,
+                tracesConfig = if (tracesEnabled) tracesConfig else null,
+                crashReportConfig = if (crashReportsEnabled) crashReportConfig else null
+            )
+        }
+
+        /**
+         * Enables or disables the logs feature.
+         * This feature is enabled by default, disabling it will prevent any logs to be sent to
+         * Datadog servers.
+         * @param enabled true by default
+         */
+        fun setLogsEnabled(enabled: Boolean): Builder {
+            logsEnabled = enabled
+            return this
+        }
+
+        /**
+         * Enables or disables the tracing feature.
+         * This feature is enabled by default, disabling it will prevent any spans and traces to
+         * be sent to Datadog servers.
+         * @param enabled true by default
+         */
+        fun setTracesEnabled(enabled: Boolean): Builder {
+            tracesEnabled = enabled
+            return this
+        }
+
+        /**
+         * Enables or disables the crash report feature.
+         * This feature is enabled by default, disabling it will prevent any crash report to be
+         * sent to Datadog servers.
+         * @param enabled true by default
+         */
+        fun setCrashReportsEnabled(enabled: Boolean): Builder {
+            crashReportsEnabled = enabled
+            return this
+        }
+
+        /**
+         * Sets the service name that will appear in your logs.
+         * @param serviceName the service name (default = "android")
+         */
+        fun setServiceName(serviceName: String): Builder {
+            logsConfig = logsConfig.copy(serviceName = serviceName)
+            tracesConfig = tracesConfig.copy(serviceName = serviceName)
+            crashReportConfig = crashReportConfig.copy(serviceName = serviceName)
+            return this
+        }
+
+        /**
+         * Let the SDK target Datadog's Europe server.
+         *
+         * Call this if you log on [app.datadoghq.eu](https://app.datadoghq.eu/).
+         */
+        fun europeEndpoints(): Builder {
```

Comment:
All other builder method use `verb` + `nouns` convention (`setXEnabled()`), maybe we should do `useEUServers()`, `useUSServers()`, `useCustomLogsServer()` and so on?

Reply:
yes indeed